checkpkg.in

```diff
@@ -5,12 +5,14 @@ shopt -s extglob
 
 m4_include(lib/common.sh)
 
+MAKEPKG_CONF=${MAKEPKG_CONF:-/etc/makepkg.conf}
+
 # Source makepkg.conf; fail if it is not found
-if [[ -r '/etc/makepkg.conf' ]]; then
+if [[ -r "$MAKEPKG_CONF" ]]; then
```

Comment:
instead of magic environment variables this seems to be a better fit for an actual CLI option like `-m`/`--makepkg-config` and move this check+source block below the option parsing. If we can switch to cli options this looks otherwise good.